src/components/data-management/ProjectsListContainer.jsx

```diff
@@ -33,7 +36,42 @@ const ProjectsListContainer = (props) => {
     border: `2px solid ${blue.primary}`,
   };
 
-  const uploadFiles = () => {
+  const uploadFiles = (filesList, sampleType) => {
```

Comment:
Both this uploadFiles and the ProjectDetails one seem to be repeated code, perhaps we could refactor this into one function they have in common? The dispatch could be passed as a parameter.

Reply:
Yea, I'd like to do that, but they just have so many store dependencies 😢